options.go

```diff
@@ -301,3 +305,41 @@ func parseVsocks(devices []string) ([]firecracker.VsockDevice, error) {
 func createFifoFileLogs(fifoPath string) (*os.File, error) {
 	return os.OpenFile(fifoPath, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
 }
+
+// Looks at directorys in order:
```

Comment:
nit: Use godoc formatting even though this is a private function (e.g., "getSocketPath looks...")